client/components/place/place-detail/place-detail.component.js

```diff
@@ -1,26 +1,33 @@
-angular.module('placeDetail', [])
+angular.module('placeDetail', ['comment'])
   .component('placeDetail', {
     templateUrl: 'components/place/place-detail/place-detail.template.html',
     bindings: {
       place: '<'
     },
-    controller: ['mapFactory', function placeDetailCtrl(mapFactory) {
+    controller: function placeDetailCtrl($scope, constants) {
+      angular.element(document).ready(function() {
+        $('.fancybox').fancybox();
+      });
       this.map = L.map('map1', {
-        center: [50.6234, 26.2189],
-        zoom: 14
+        center: constants.mapCenter,
+        zoom: constants.defaultZoom-6
       });
-      Streets = L.tileLayer('http://{s}.tile.osm.org/{z}/{x}/{y}.png', {
+      this.map.touchZoom.disable();
+      this.map.dragging.disable();
+      this.map.scrollWheelZoom.disable();
+      var layerStreet = L.tileLayer('http://{s}.tile.osm.org/{z}/{x}/{y}.png', {
         attribution: '&copy; <a href="http://osm.org/copyright">OpenStreetMap</a> contributors'
       });
-      this.map.addLayer(Streets);
-      this.location = this.place.getLocation();
-      this.marker = L.marker(this.location).addTo(this.map);
-
-      this.marker.bindPopup('<div><h3>' + this.place.name +
-        '</h3><a><img class="marker-image" src="assets/' + this.place.photo[0] +
-        '" /></a><br />').openPopup();
+      this.map.addLayer(layerStreet);
+      this.marker = L.marker(L.latLng(this.place.location.coordinates[1], this.place.location.coordinates[0])).addTo(this.map);
+      this.marker.bindPopup('<div class="popup"><h3>' + this.place.name +
+        '</h3><a><img class="marker-image center-block" src="' + this.place.photos[0] + '" /></a><br />').openPopup();
+      var deltaheight = 0.1;
+      this.map.setView([this.place.location.coordinates[1]+ deltaheight, this.place.location.coordinates[0]]); //0.1- for responcive design- show info on mobile
```

Comment:
Make this line smaller than 80 characters and remove comment from there (if you want comment to stay, move it to line 25, where `deltaheight` variable is declared)